src/libraries/System.Net.Http/src/System/Net/Http/BrowserHttpHandler/BrowserHttpHandler.cs

```diff
@@ -95,14 +99,26 @@ public bool PreAuthenticate
 
         public bool AllowAutoRedirect
         {
-            get => throw new PlatformNotSupportedException();
-            set => throw new PlatformNotSupportedException();
+            get => _allowAutoRedirect;
+            set
+            {
+                _isAllowAutoRedirectTouched = true;
```

Comment:
I would swap these. You are reading them in the same order in `SendAsync`. Depending on whether multi-threading is a thing in wasm, you could read `_isAllowAutoRedirectTouched` and still use the original `_allowAutoRedirect` value (before it's changed here).

Reply:
Addressed in recent commits.